test/util/tools/SceApiTester.js

```diff
@@ -0,0 +1,57 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+chai.should();
+chai.use(chaiHttp);
+
+class SceApiTester{
+  constructor(app){
+    this.app = app;
+  }
+
+  async sendPostRequest(endpoint, params){
```

Comment:
consider adding documentation to this class, e.g.
1. [`api/util/PathParser.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/util/PathParser.js)
1. [`api/SceHttpServer.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/SceHttpServer.js)
1. [`api/mailer/SceGoogleApiHandler.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/mailer/SceGoogleApiHandler.js)